src/test/java/com/powsybl/study/server/StudyTest.java

```diff
@@ -97,6 +97,18 @@ public void setup() {
         studyService.setSingleLineDiagramServerRest(singleLineDiagramServerRest);
         studyService.setGeoDataServerRest(geoDataServerRest);
 
+        given(caseServerRest.exchange(
```

Comment:
This one is not used?

Reply:
it's used

Reply:
studyTest when inserting a study we should call that method : line 188

Reply:
ok